.github/workflows/markdown-linting/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "markdown-linting",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "fix:markdownlint": "markdownlint --fix ../../../resources",
+    "lint:markdownlint": "markdownlint ../../../resources"
```

Comment:
How about putting these at the top of the repo (in this PR or later)? Having  `markdownlint-cli` and `.markdownlint.json` there will let folks use https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint and thus see the issues before a commit is made.

Reply:
Agree with this, and I also think we might as well also format / lint all the markdown in the repository rather than just the resources folder.

Can you also please use `yarn` to install rather than `npm`? So we don't end up with a mixture of lock files in the repo - we'll be bringing in `yarn.lock`s from hash.dev. Can you also create a task to install with `yarn` for the algolia action too in a separate PR? We have a `package-lock.json` over there.

Reply:
This was discussed on a call with Alex yesterday, since we don't want a `package.json` at the root of our repo, it's fine to keep it here for now.

Reply:
This should be fixed now